src/Evolution/DiscontinuousGalerkin/NormalVectorTags.hpp

```diff
@@ -0,0 +1,35 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <optional>
+#include <unordered_map>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Domain/Structure/Direction.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// Tags used on the interior faces of the elements
+namespace evolution::dg::Tags::InternalFace {
+/// The magnitude of the unnormalized normal covector to the interface
+struct MagnitudeOfNormal : db::SimpleTag {
+  using type = Scalar<DataVector>;
+};
+
+/// The normal covector to the interface
+template <size_t Dim>
+struct NormalCovector : db::SimpleTag {
+  using type = tnsr::i<DataVector, Dim, Frame::Inertial>;
+};
+
+template <size_t Dim>
+struct NormalCovectorQuantities : db::SimpleTag {
```

Comment:
Do you plan to add more quantities to this later? If not, then is there a reason Therese are in a Variables instead of a std::pair or std::array or something?

Also, if you don't plan to add more quantities later, why not NormalMagnitudeAndOneForm or, if you stick with convector, NormalMagnitudeAndCovector, to avoid the generic "quantities" name that isn't as informative?

Reply:
The main reason for combining the tags is to use a Variables so that there's only one memory allocation for this computation.

I think renaming it is fine. Initially I wasn't sure what all I would need, but at this point I'm reasonably confident that there won't be anything else.